OPENJDK-2408-bin-custom-s2i-assemble/.s2i/bin/assemble

```diff
@@ -8,4 +8,5 @@ echo appsrc-provided s2i assemble script executed
 sleep 1
 echo delegating to builder image s2i assemble
 
-exec /usr/local/s2i/assemble "$@"
+PATH=/usr/libexec/s2i:/usr/local/s2i:$PATH
```

Comment:
This might warrant a comment mentioning why both paths need to be added.

Reply:
Sure OK